server/app/views/applicant/ProgramIndexView.java

```diff
@@ -106,20 +108,62 @@ public Content render(
     bundle.setTitle(messages.at(MessageKey.CONTENT_GET_BENEFITS.getKeyName()));
     bannerMessage.ifPresent(bundle::addToastMessages);
     bundle.addMainContent(
-        topContent(
-            messages.at(MessageKey.CONTENT_GET_BENEFITS.getKeyName()),
-            messages.at(
-                MessageKey.CONTENT_CIVIFORM_DESCRIPTION.getKeyName(), civicEntityShortName)),
+        topContent(messages, userName),
         mainContent(
             request, messages, applicationPrograms, applicantId, messages.lang().toLocale()));
 
     return layout.renderWithNav(request, userName, messages, bundle, /*includeAdminLogin=*/ true);
   }
 
-  private DivTag topContent(String titleText, String infoTextLine) {
-    // "Get benefits"
+  private DivTag topContent(Messages messages, Optional<String> userName) {
+    String displayUserName = ApplicantUtils.getApplicantName(userName, messages);
+
+    String guestUserName = messages.at(MessageKey.GUEST.getKeyName());
+
+    String h1Text, infoDivText, widthClass;
+    DivTag buttonsDiv;
+
+    if (displayUserName.equals(guestUserName)) {
+      // "Save time when applying for benefits"
+      h1Text = messages.at(MessageKey.CONTENT_SAVE_TIME.getKeyName());
+      infoDivText =
+          messages.at(MessageKey.CONTENT_GUEST_DESCRIPTION.getKeyName(), civicEntityShortName);
+      widthClass = "w-8/12";
+      // Log in and Create account buttons if user is a guest.
+      buttonsDiv =
+          div()
+              .with(
+                  redirectButton(
+                          "login-button",
+                          messages.at(MessageKey.BUTTON_LOGIN.getKeyName()),
+                          routes.LoginController.applicantLogin(Optional.empty()).url())
+                      .withClasses(ApplicantStyles.BUTTON_PROGRAMS_PAGE_WHITE, "basis-60"))
+              .with(
+                  redirectButton(
+                          "create-account",
+                          messages.at(MessageKey.BUTTON_CREATE_ACCOUNT.getKeyName()),
+                          routes.LoginController.register().url())
+                      .withClasses(ApplicantStyles.BUTTON_PROGRAMS_PAGE_WHITE, "basis-60"))
+              .withClasses(
+                  "flex",
+                  "flex-row",
+                  "gap-x-8",
+                  "pb-6",
+                  "px-4",
+                  "w-screen",
+                  "place-content-center");
+    } else { // Logged in.
+      // "Get benefits"
+      h1Text = messages.at(MessageKey.CONTENT_GET_BENEFITS.getKeyName());
+      infoDivText =
+          messages.at(MessageKey.CONTENT_CIVIFORM_DESCRIPTION.getKeyName(), civicEntityShortName);
+      widthClass = "w-5/12";
+      // No buttons if logged in.
+      buttonsDiv = div();
```

Comment:
**Suggestion (non-blocking):** Rather than inserting an empty div, might be better to make this optional

Reply:
Done.